src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+    public String value;
+    public boolean isPrivate;
+
+    public Contact() {
+        super();
```

Comment:
this call to super will call `Object` class, and does nothing; so may be you can remove this!